docs/source/contents/4_draft_mapping.rst

```diff
@@ -124,21 +124,53 @@ XPath
 ^^^^^
 
 :code:`identifier[@type="Vendor ID"]` OR
+
+
 :code:`identifier[@type="archival number"]` OR
+
+
 :code:`identifier[@type="catalog"]` OR
+
+
 :code:`identifier[@type="circular"]` OR
+
+
 :code:`identifier[@type="cw"]` OR
+
+
 :code:`identifier[@type="document ID"]` OR
+
+
 :code:`identifier[@type="documentID"]` OR
+
+
 :code:`identifier[@type="filename"]` OR
+
+
 :code:`identifier[@type="film number"]` OR
+
+
 :code:`identifier[@type="legacy"]` OR
+
+
 :code:`identifier[@type="local"]` OR
```

Comment:
Dead serious:  why double returns?

Reply:
PEBKAC. I... double-applied a regex? Honestly I'm not sure how I screwed this up, but thanks for asking: I found another screwup with your help.